src/db/handlers/mod.rs

```diff
@@ -0,0 +1,22 @@
+use crate::db::messages;
+use crate::db::models;
+use crate::db::DbExecutor;
+use crate::schema;
+use actix_web::actix::*;
+use actix_web::error;
+use diesel;
+use diesel::prelude::*;
+use diesel::r2d2::{ConnectionManager, Pool};
+
+impl Handler<messages::GetUser> for DbExecutor {
+    type Result = Result<Option<models::User>, actix_web::Error>;
+    fn handle(&mut self, msg: messages::GetUser, ctx: &mut Self::Context) -> Self::Result {
+        use crate::schema::users::dsl::*;
```

Comment:
@damon-myers this is where we import the generated schema types. They create a DSL for interacting with fields e.g.
```
let mut items = users
            .filter(username.eq(msg.username.clone()))
```